src/main/java/io/tiledb/spark/TileDBDataReaderPartitionScan.java

```diff
@@ -4,6 +4,9 @@
 import static io.tiledb.java.api.QueryStatus.TILEDB_COMPLETED;
 import static io.tiledb.java.api.QueryStatus.TILEDB_INCOMPLETE;
 import static io.tiledb.java.api.QueryStatus.TILEDB_UNINITIALIZED;
+import static io.tiledb.libtiledb.tiledb_query_condition_combination_op_t.TILEDB_AND;
```

Comment:
nit: this should be enum under `io.tiledb.java.api.QueryCondition`, right?

(ok to leave for follow-up though)

Reply:
Not sure what you mean

Reply:
I mean that the other enums are translated from `tiledb_*_*_t` to something like `io.tiledb.java.api.QueryStatus.TILEDB_UNINITIALIZED` or `QueryType.TILEDB_READ` so we should do the same here for consistency -- doesn't need to be in this PR though.

Reply:
Nevermind, looks like `QueryStatus` is the only one like this.